### DISCUSSION TITLE
New Custom Dashboards

### DISCUSSION THREAD

**Discussion by CarlosCuezva** [teslamate-org/teslamate#3085] [Show and tell]
Hello everyone!

I´ve submitted a PR (#3071) to integrate some new dashboards to squeeze even more data from our cars.

Whether it's approved or not, here's my repository for you to try if you want.

https://github.com/CarlosCuezva/dashboards-Grafana-Teslamate

I hope you like them

---

### A few examples

![image](https://user-images.githubusercontent.com/5815421/214540115-dc40eb52-9bc3-44da-bd10-8e961a8e2899.png)

![image](https://user-images.githubusercontent.com/5815421/214540217-058c2055-9012-4592-b350-e048a2e1dd4c.png)

![image](https://user-images.githubusercontent.com/5815421/214540410-e3dd3ddf-eb93-4d09-acde-460a376c8772.png)

**Comment by tomS3210** [comment]
Thank you for you dashboards, they are now installed on my Teslamate :-)

**Reply by CarlosCuezva** [reply]
I'm glad that you liked it ;)

**Comment by CarlosCuezva** [comment]
Updated dashboards to version **1.21.1** with these changes:

- Battery Health: Optimized queries
- Charges: Added color scale to recharge efficiency
- Tire Pressure: Revised queries so that they can't return a null value

**Comment by CarlosCuezva** [comment]
Updated dashboards to version **1.22.0** with these changes:

- Battery Health: Improved degradation calculation
- Battery Health: Removed timepicker
- Battery Health: Display of the degradation curve from the first data not between time periods
- Battery Health: Text changes
- Tire Pressure: Fixed bug when converting pressure to PSI
- Charging Costs Stats: Fixed calculation of total months
